server/game/gamechat.js

```diff
@@ -22,8 +22,8 @@ class GameChat {
         args = _.reduce(args, (argList, arg) => {
             if(arg instanceof Spectator) {
                 argList.push(arg.name);
-            } else if(arg && arg.emailHash) {
-                argList.push({ name: arg.name, emailHash: arg.emailHash, noAvatar: arg.user.settings.disableGravatar });
+            } else if(arg && arg.name) {
+                argList.push({ name: arg.name });
```

Comment:
I get the feeling this will try to format Card arguments as players / avatars, since card objects also have `name`. It might be useful to also add some sort of indicator for the client that it should format this as a name + avatar, rather than just as a object with name. E.g.
```javascript
argList.push({ name: arg.name, argType: 'player' });
```

Reply:
Having thought a bit more, it may also be useful to send the URL to the client directly, especially if the rules about transforming usernames to valid filenames (strip this character out, escape this character, handling case sensitivity, etc) becomes complex.

```javascript
argList.push({ name: arg.name, avatarUrl: arg.avatarUrl, argType: 'player' });
```